### ISSUE TITLE
"due date" picker one day wrong

### ISSUE THREAD

**Issue by StefaanVanDooren** [osTicket/osTicket-1.7#834]
code patched with #833 

When selecting a date with the datepicker (date format set to "d/m/Y"), the date is set one day behind.
Selecting 27/11/2013 will be 26/11/2013 after saving.

**Comment by greezybacon** [comment]
I think I give up. Perhaps we should reinstate the original shortcoming

**Comment by greezybacon** [comment]
I tried this morning and I can't reproduce it. I tried changing my system timezones around (in the admin panel). Is the time changing too? (ie. if you select 27/11/2013 0:00, does it switch to 26/11/2013 21:00 or something similar?) Do you know if you were getting this particular behavior on code before these changes (ie. v1.7.2 stock)?

**Comment by StefaanVanDooren** [comment]
Time stays as it should. Before the patch everything was working as expected. only the format in editing mode was wrong.
I noticed that when I hit save, the date  goes from "27/11/2013" to "2013-11-26" and then "26/11/2013"

Keep in mind I have to hit "save" or "open" before the date changes.

I attached another screenshot with my format settings

![settings](https://f.cloud.github.com/assets/5485451/1453837/52de462c-42fd-11e3-99b1-76fc8e761e31.png)
